docs/rules/callback-return.md

```diff
@@ -24,14 +24,16 @@ preceding a `return` statement. This rule decides what is a callback based on th
 
 ## Options
 
-The rule takes a single option, which is an array of possible callback names. The default callback names are `callback`, `cb`, `next`.
+The rule takes a single option - an array of possible callback names - which may include object methods. The default callback names are `callback`, `cb`, `next`.
+
+### Default callback names
 
 Examples of **incorrect** code for this rule with the default `["callback", "cb", "next"]` option:
 
 ```js
 /*eslint callback-return: "error"*/
 
-function foo() {
+function foo(err, callback) {
```

Comment:
For consistency with the example at the beginning. I think it makes the examples a little more concrete - please let me know if you disagree.